### PR TITLE
Editorial: Add informative section about ARIA semantics that extend/diverge from HTML

### LINKED ISSUE #294: create informative section for ARIA vs HTML allowances
further guidance on why ARIA allows for certain attributes to be used with roles, where using those same attributes on the HTML element with the implicit ARIA semantics is not allowed.

### PR DESCRIPTION
**Pull request by patrickhlauke**
Closes https://github.com/w3c/html-aria/issues/294

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/html-aria/pull/305.html" title="Last updated on Dec 5, 2021, 10:18 PM UTC (883bb38)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/html-aria/305/e2b48a3...883bb38.html" title="Last updated on Dec 5, 2021, 10:18 PM UTC (883bb38)">Diff</a>

### CODE REVIEW

**Comment by scottaohara** [comment]
thanks for getting this started @patrickhlauke. i'm going to pull it down and take a stab at some other parts i wanted to get into here.

**Comment by patrickhlauke** [comment]
Sure thing, have at it. This was my first brain-dump, happy to have it tweaked further.

**Comment by patrickhlauke** [comment]
@scottaohara pinging you to see if you wanted to add anything more, or if we should just merge?

**Comment by scottaohara** [comment]
On my list but getting the stuff we talked about today done first

**Review by scottaohara — APPROVED** [review]

**Comment by scottaohara** [comment]
on a long enough timeline, the thought of making additional edits apparently tracks down to zero (as I cannot remember for the life of me what else I had in mind, back in April).

Let's get this merged once the spec goes to rec.   (just talked to marcos and we really shouldn't push any non-essential editorial at this time)

File: index.html [hunk 1]
```diff
@@ -88,6 +88,20 @@ <h2 id="rules-wd">
...
       </p>
     </section>
+    <section class="informative">
+      <h2>ARIA semantics that extend/diverge from HTML</h2>
+      <p>Through the use of ARIA, authors can express semantics that go beyond the current native
+        HTML capabilities. As a result, there are situations where certain `aria-*` attributes are
+        allowed for use on elements with particular `role`s, while the equivalent native attribute
+        is currently not allowed in HTML itself.</p>
+      <p>For instance, HTML has no direct concept of a disabled link. Constructs such as `&lt;a href="..." disabled&gt; ... &lt;/a&gt;` are not valid, and will not be conveyed to assistive technologies. However, using ARIA, it is valid to
+        define a disabled link using `&lt;a href="..." aria-disabled="true"&gt; ... &lt;/a&gt;`. While user agents
```
Comment (marcoscaceres) [inline index.html]: Just wondering... is there an issue filed for this in HTML? Seems kinda nice to do:

```
<a href="" disabled>
```

As an equivalent to using `aria-disabled="true"`

Comment (scottaohara) [reply index.html]: this has been discussed... it really is just bad authoring and, imo, we should not be advocating for bad authoring.

Comment (marcoscaceres) [reply index.html]: Wait, aria-disabled="true" is bad? I'm confused?

Comment (scottaohara) [reply index.html]: No, I meant trying to make a link disabled specifically.

Comment (marcoscaceres) [reply index.html]: Ah, got it. Yes, though I imagine people do it. Sorry, another silly question: how come it’s valid with aria-? Shouldn’t it also be invalid?

Comment (patrickhlauke) [reply index.html]: i wouldn't be adverse to seeing `disabled` on links in HTML spec as well. i mean the "correct" current practice is to turn the link into another element or at the very least removing the `href` to make it a placeholder/anchor link. i could imagine spec'ing that if a link has a native `disabled` attribute that the browser should treat it as such / ignore the `href` and neutralise the default action for it

**Comment by scottaohara** [comment]
@patrickhlauke made a PR for some additional ideas for your PR.  (apparently on a _longer_ enough timeline, I was able to remember the things I had forgotten I had wanted to add to your PR).

**Review by stevefaulkner — APPROVED** [review]

**Pull request merged**